src/applications/personalization/profile/tests/components/direct-deposit/BankInfoCNP.unit.spec.jsx

```diff
@@ -77,9 +77,12 @@ function fillOutAndSubmitBankInfoForm(view) {
 }
 
 function findSetUpBankInfoButton(view) {
-  return view.queryByText(/please add your bank information/i, {
-    selector: 'button',
-  });
+  return view.queryByLabelText(
+    /Add your direct deposit for disability compensation and pension benefits bank information/i,
```

Comment:
Using a highly specific label like this is totally fine and maybe even better than my usual approach of keeping selector labels kinda fuzzy. For example, I would probably do something like `/add.*disability.*bank info/i`. My reason for that is that it allows UX/design to change the label a little bit and we won't get a failing test. But I totally get why using a very specific label makes sense so I don't have a strong preference either way.

Reply:
I had originally wrote these selectors when I was very new to RTL, but if I did it again today I'd use `view.queryByRole('button', {name: /regex here/i})` instead of `view.queryByText()`. But this is totally fine.

Reply:
I will do that. I just have to figure out how to use label for it. Just checking for a single button that says edit was clashing with something which is why I went with the label since it's more specific